soc/silabs/silabs_siwx91x/siwg917/nwp.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2024 Silicon Laboratories Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef SIWG917_NWP_H
+#define SIWG917_NWP_H
+
+#include "sl_wifi.h"
+
+static inline int get_wifi_device_configuration(sl_wifi_device_configuration_t *net_dev_cfg)
```

Comment:
Why this function is defined in the header file?

BTW, prefix the function name with the right prefix (`siwg917_nwp_` I think).